packages/framework/assets/js/admin/components/AdvancedSearch.js

```diff
@@ -70,7 +70,7 @@ export default class AdvancedSearch {
     }
 
     static updateValueByOperator ($rulesContainer, $rule, operator) {
-        $rule.find('.js-advanced-search-rule-value').toggle(operator !== 'notSet');
+        $rule.find('.js-advanced-search-rule-value').toggle(operator !== 'notSet' && operator !== 'notRegistered');
```

Comment:
Just a note, pls do not fix it. 

It would be nice to configure this on PHP level, so I don't need to override JS on project every time I add such an operator

Reply:
The  operator `notSet` has many times same functionality, but translation can differ.